VS 21 February/VS 21 February/CreditCard.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+namespace VS_21_February
+{
+	public class CreditCard
+	{
+        public string accountNumber;
+        public double currentSumm;
+
+        public CreditCard()
+        {
+
+        }
+
+        public double AccrualMoney(double summOfMoney)
+        {
+            currentSumm += summOfMoney;
+            return currentSumm;
+        }
+
+        public double WithdrawalMoney(double summOfMoney)
```

Comment:
When withdrawing money, the method should print how much
banknotes of what denomination the amount is issued.